toolchain/cc_toolchain_config.bzl

```diff
@@ -129,15 +129,15 @@ def _impl(ctx):
               actions = all_compile_actions,
               flag_groups = [
                   flag_group(
-                      flags = [
+                      flags = ctx.attr.extra_compiler_flags + [
```

Comment:
@dlj-NaN Does this fix the ordering problem?

Reply:
Yeah, that should solve the include path search issue.

We might still want the extra_compiler_flags to be ordered after the the warning/functional defaults (`-Wall` and `-fvisibility`), though. That way, we can control warnings/visibility via `extra_compiler_flags` as well as include path injection. It might make sense just to split into three flag_groups:

1. everything else,
2. `ctx.attr.extra_compiler_flags`, and
3. the toolchain include path.

Reply:
Ok I think I fixed it, can you take another look?